mteb/tasks/Classification/swa/SwahiliNewsClassification.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+# from ....abstasks import AbsTaskClassification
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SwahiliNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SwahiliNewsClassification",
+        description="Dataset for Swahili News Classification, categorized with 5 domains. Building and Optimizing Swahili Language Models: Techniques, Embeddings, and Datasets",
+        reference="https://huggingface.co/datasets/Mollel/SwahiliNewsClassification",
```

Comment:
Seems like the dataset has quite limited documentation?

Reply:
I have added to the HF the documentation of dataset

Reply:
Thanks @msamwelmollel much appreciated